user_docs/en/userGuide.t2t

```diff
@@ -2624,28 +2658,18 @@ If the add-on was previously "disabled", a status will show "enabled after resta
 If the add-on was previously "enabled", a status will show "disabled after restart".
 Just like when you install or remove add-ons, you need to restart NVDA in order for changes to take effect.
 
-The manager also has a Close button to close the dialog.
-If you have installed, removed or changed the status of an add-on, NVDA will first ask you if you wish to restart so that your changes can take effect.
-
++++ Incompatible Add-ons +++[incompatibleAddonsManager]
 Some older add-ons may no longer be compatible with the version of NVDA that you have.
 When using an older version of NVDA, some new add-ons may not be compatible either.
 Attempting to install an incompatible add-on will result in an error explaining why the add-on is considered incompatible.
-To inspect these incompatible add-ons, you can use the "view incompatible add-ons" button to launch the incompatible add-ons manager.
-
-To access the Add-ons Manager from anywhere, please assign a custom gesture using the [Input Gestures dialog #InputGestures].
-
-+++ Incompatible Add-ons Manager +++[incompatibleAddonsManager]
-The Incompatible Add-ons Manager, which can be accessed via the "view incompatible add-ons" button in the Add-on manager, allows you to inspect any incompatible add-ons, and the reason they are considered incompatible.
-Add-ons are considered incompatible when they have not been updated to work with significant changes to NVDA, or when they rely on a feature not available in the version of NVDA you are using.
-The Incompatible add-ons manager has a short message to explain its purpose as well as the version of NVDA.
-The incompatible add-ons are presented in a list with the following columns:
-+ Package, the name of the add-on
-+ Version, the version of the add-on
-+ Incompatible reason, an explanation of why the add-on is considered incompatible
-+
 
-The Incompatible add-ons manager also has an "About add-on..." button.
-This dialog will provide you with the full details of the add-on, which is helpful when contacting the add-on author.
+For older add-ons, you can override the incompatibility at your own risk.
+Incompatible add-ons may not work with your version of NVDA, and can cause bugs including crashing.

```

Comment:
I'd probably say "can cause problems" rather than bugs here.  A bug implies something not working as intended, for which you might file an issue.  You could certainly file an issue on the add-on's page asking them to update it, and including what isn't working is helpful - but it's not that it's not working as intended in that case, it's that it was never intended to work with the version of NVDA you are using.

Reply:
how about "can cause unstable behaviour"